ticdc/manage-ticdc.md

```diff
@@ -596,6 +596,16 @@ For nodes other than owner nodes, executing the above command will return the fo
 }
 ```
 
+### Dynamically change the log level of TiCDC server
+
+{{< copyable "shell-regular" >}}
+
+```shell
+curl -X POST -d '"debug"' http://127.0.0.1:8301/admin/log
+```
+
+In the command above, the `POST` parameter indicates the new log level. The [zap-provided](https://godoc.org/go.uber.org/zap#UnmarshalText) log level options are supported: "debug", "info", "warn", "error", "dpanic", "panic", and "fatal". This interface parameter is JSON-encoded and you need to pay attention to the use of quotation marks. For example: `"debug"`.
```

Comment:
The original [Chinese version](https://github.com/pingcap/docs-cn/pull/4560/files#diff-c838c41b0017f0b0cb1d5bb0a42dcfccR594) is ``` `'"debug"'` ```, which one is wrong? @TomShawn @amyangfei

Reply:
should be `` `'"debug"'` ``

Reply:
Fixed in https://github.com/pingcap/docs/pull/4005/commits/68c5561ff4eef1c30bd27c0c64093ff3bee36103.